filebeat/docs/filebeat-options.asciidoc

```diff
@@ -1,544 +1,68 @@
-backed off multiple times. The default is 10s.
-
-Requirement: max_backoff should always be set to `max_backoff <= scan_frequency`. In case `max_backoff` should be bigger, it is recommended to close the file handler instead let the prospector pick up the file again.
-
-[float]
-==== `backoff_factor`
-
-This option specifies how fast the waiting time is increased. The bigger the
-backoff factor, the faster the `max_backoff` value is reached. The backoff factor
-increments exponentially. The minimum value allowed is 1. If this value is set to 1,
-the backoff algorithm is disabled, and the `backoff` value is used for waiting for new
-lines. The `backoff` value will be multiplied each time with the `backoff_factor` until
-`max_backoff` is reached. The default is 2.
-
-[float]
-[[harvester-limit]]
-==== `harvester_limit`
-
-The `harvester_limit` option limits the number of harvesters that are started in parallel for one prospector. This directly relates
-to the maximum number of file handlers that are opened. The default for `harvester_limit` is 0, which means there is no limit. This
-configuration is useful if the number of files to be harvested exceeds the open file handler limit of the operating system.
-
-Setting a limit on the number of harvesters means that potentially not all files are opened in parallel. Therefore we recommended that you use
-this option in combination with the `close_*` options to make sure harvesters are stopped more often so that new files can be
-picked up.
-
-Currently if a new harvester can be started again, the harvester is picked randomly. This means it's possible that the harvester for a file that was just closed and then updated again might be started instead of the harvester for a file that hasn't been harvested for a longer period of time.
-
-This configuration option applies per prospector. You can use this option to indirectly set higher priorities on certain prospectors
-by assigning a higher limit of harvesters.
-
-[float]
-==== `enabled`
-
-The `enabled` option can be used with each prospector to define if a prospector is enabled or not. By default, enabled is set to true.
-
-[float]
-[[max-message-size]]
-==== `max_message_size`
-
-When used with `type: udp`, specifies the maximum size of the message received over UDP. The default is 10240.
-
-[float]
-[[config-containers]]
-==== `containers`
+You can configure {beatname_uc} to use the following inputs:
 
-experimental[]
+// REVIWERS: I would like to make the following list generated, if possible. Can
+// we reuse the logic that we use for generating the modules list?
 
-These options are only available when using `docker` prospector type. They allow to configure the list of containers to read logs from.
+* <<log-input>>
+* <<stdin-input>>
+* <<redis-input>>
+* <<udp-input>>
+* <<docker-input>>
```

Comment:
How about renaming the anchors to `filebeat-input-<service>`. Also name the files `input-<service>.asciidoc`. So we have some grouping and reduce chance of naming conflicts.